src/Infrastructure/RulesContext.cs

```diff
@@ -493,6 +494,22 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             entity.Property(e => e.Date).HasColumnName("created_at");
         });
 
+        modelBuilder.Entity<ConversationHistoryModel>(entity =>
```

Comment:
Consider extracting EF configuration into a separate class.

Reply:
I've raised an issue to track this: https://github.com/SSWConsulting/SSW.Rules.GPT/issues/151